pkg/dashboard/dashboard.go

```diff
@@ -52,18 +66,133 @@ func Dashboard(opts Options) http.Handler {
 			return
 		}
 
+		if costPerCPU != "" && costPerGB != "" {
+			costPerCPUFloat, _ := strconv.ParseFloat(costPerCPU, 64)
+			costPerGBFloat, _ := strconv.ParseFloat(costPerGB, 64)
+
+			var containerCost, guaranteedCost, burstableCost float64
+
+			for _, n := range vpaData.Namespaces {
+				for _, w := range n.Workloads {
+					for k, c := range w.Containers {
+						containerCost = calculateContainerCost(costPerCPUFloat, costPerGBFloat, c)
+						guaranteedCost, burstableCost = calculateRecommendedCosts(costPerCPUFloat, costPerGBFloat, containerCost, c)
+						c.ContainerCost = containerCost
+						c.ContainerCostInt = getCostInt(containerCost)
+						c.GuaranteedCostInt = getCostInt(guaranteedCost)
+						c.BurstableCostInt = getCostInt(burstableCost)
+						c.GuaranteedCost = math.Abs(guaranteedCost)
+						c.BurstableCost = math.Abs(burstableCost)
+						w.Containers[k] = c
+					}
+				}
+			}
+		}
+
 		tmpl, err := getTemplate("dashboard",
 			"container",
 			"dashboard",
 			"filter",
 			"namespace",
+			"email",
+			"api_token",
+			"cost_settings",
 		)
 		if err != nil {
 			klog.Errorf("Error getting template data %v", err)
 			http.Error(w, "Error getting template data", http.StatusInternalServerError)
 			return
 		}
 
-		writeTemplate(tmpl, opts, &vpaData, w)
+		data := struct {
+			VpaData      summary.Summary
+			InsightsHost string
+		}{
+			VpaData:      vpaData,
+			InsightsHost: opts.insightsHost,
+		}
+
+		writeTemplate(tmpl, opts, &data, w)
 	})
 }
+
+func calculateContainerCost(costPerCPUFloat float64, costPerGBFloat float64, c summary.ContainerSummary) float64 {
+	var cpuRequests, memRequests, cpuLimits, memLimits float64
+
+	cpuRequests = 0
+	memRequests = 0
+	cpuLimits = 0
+	memLimits = 0
```

Comment:
I believe the zero-value of a float in go is zero, and it's not nillable, so these are unnecessary after declaring the vars.